### ISSUE TITLE
import jar from Maven coordinates

### ISSUE THREAD

**Issue by vietj** [ceylon/ceylon-compiler#2401]
would be nice to be able to do something like:

```
ceylon import-jar maven:io.vertx:vertx-core:3.1.0.jar
```

supporting maven coordinates (including classifiers)

that would avoid to have to copy them locally from a maven repository first.

**Comment by FroMage** [comment]
Yes, and it would use the `pom.xml` dependencies as well, I guess, to produce a `module.xml` file for it.

The question is should it add it in the jar too? Possibly also generate the OSGi info?

**Comment by vietj** [comment]
it should be possible to override module.xml and provide it like the current tool is doing
